### PR TITLE
Added the Introduction to the Information Security and Quality Assura…

### LINKED ISSUE #35550: Fill in introduction sections
Some introduction sections still haven't been created.

**PR Created**
- [x] [Intermediate Algorithm Scripting](https://www.freecodecamp.rocks/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/) - @thecodingaviator #34405 
- [x] [Information Security and Quality Assurance Projects](https://www.freecodecamp.rocks/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/) - @Sumer16 #35576 
- [x] [Take Home Projects](https://www.freecodecamp.rocks/learn/coding-interview-prep/take-home-projects/)

**PR Merged**
- [x] [Intermediate Algorithm Scripting](https://www.freecodecamp.rocks/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/)
- [x] [Information Security and Quality Assurance Projects](https://www.freecodecamp.rocks/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/)
- [ ] [Take Home Projects](https://www.freecodecamp.rocks/learn/coding-interview-prep/take-home-projects/)

### PR DESCRIPTION
**Pull request by Sumer16**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Refer #35550

### CODE REVIEW

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
 ---
 ## Introduction to the Information Security and Quality Assurance Projects
 
-This introduction is a stub
+So, now it’s time to put your newly learnt skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```
Comment (thecodingaviator) [inline client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]:
```suggestion
Now it’s time to put your newly learnt skills to work! By working on these projects, you would have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
...
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+In this section you get the chance to:
+* Build a Metric-Imperial Converter
+* Build an Issue Tracker
+* Build a Personal Library
+* Build a Stock Price Checker
+* Build an Anonymous Message Board
+
+By the end of this, you will have five IS & QA projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (thecodingaviator) [inline client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]:
```suggestion
When you are done, you will have five Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
...
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+In this section you get the chance to:
+* Build a Metric-Imperial Converter
+* Build an Issue Tracker
+* Build a Personal Library
+* Build a Stock Price Checker
+* Build an Anonymous Message Board
+
+When you are done, you will have five Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (thecodingaviator) [inline client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: Do we want to list the number of challenges over here>

cc: @RandellDawson @moT01

Comment (moT01) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: It says there's five projects, I don't like to be repetitive - what did you have in mind?

Comment (thecodingaviator) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: I want to get the number removed completely. See https://github.com/freeCodeCamp/freeCodeCamp/pull/34405#discussion_r264282065

Comment (moT01) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: Sounds good to me either way, but I suppose that's a good point - if the number ever changes, we don't have to modify these.

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md (hunk 1 above)
Comment (RandellDawson) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: Please don't use the word `learnt`.  That is British English and it should be the American English word `learned`.

Comment (Sumer16) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: Yes, I have made the cahnges.

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md (hunk 3 above)
Comment (thecodingaviator) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: @Sumer16 Go ahead and reword this so it doesn't show no of challenges

**Comment by Sumer16** [comment]
@thecodingaviator Check this out.

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md (hunk 1 above)
Comment (thecodingaviator) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: I don't see the changes though

Comment (Sumer16) [reply client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]: I updated the file can you check this one? @thecodingaviator

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md [hunk 4]
```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
...
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+In this section you get the chance to:
+* Build a Metric-Imperial Converter
+* Build an Issue Tracker
+* Build a Personal Library
+* Build a Stock Price Checker
+* Build an Anonymous Message Board
+
+When you are done, you will have plenty of Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (lynxlynxlynx) [inline client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]:
```suggestion
When you are done, you will have plenty of Information Security & Quality Assurance projects under your belt along with a certification that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

File: client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md [hunk 5]
```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
 ---
 ## Introduction to the Information Security and Quality Assurance Projects
 
-This introduction is a stub
+Now it’s time to put your newly learned skills to work! By working on these projects, you would have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```
Comment (lynxlynxlynx) [inline client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md]:
```suggestion
Now it’s time to put your newly learned skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
```

**Review by lynxlynxlynx — APPROVED** [review]
LGTM!

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
@Sumer16 Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by Sumer16** [comment]
Thank you @thecodingaviator for approving my PR.

**Pull request merged**